app/src/main/java/io/multy/storage/DatabaseHelper.java

```diff
@@ -161,4 +161,9 @@ public CurrenciesRate getCurrenciesRate() {
     public WalletAddress getWalletAddress(int id) {
         return realm.where(WalletAddress.class).equalTo("index", id).findFirst();
     }
+
+    public void removeWallet(int id) {
+        WalletRealmObject wallet = getWalletById(id);
```

Comment:
It's good practice to get realm models inside transaction.
So execute transaction first
and then make getWalletById

execute -> getWallet(id).delete();

see realm documentation for further details